spec/features/pilot/step_1_understand_service_spec.rb

```diff
@@ -12,9 +12,10 @@
   #   And I should see what information I will need to provide
   #   When I choose to use the service
   #   Then I should find myself on the first step of the service
-  scenario "Visit the about page" do
+  scenario "Step 1 - pilot reviews information about service" do
```

Comment:
The spec suggests this is all still step 1, but the commit message suggests this is step 2, or does the `ii` in the commit message mean `1ii`?